tsconfig.json

```diff
@@ -2,7 +2,8 @@
   "compilerOptions": {
     "target": "es6",
     "lib": [
-      "esnext"
+      "esnext",
+      "dom"
```

Comment:
would it be possible to remove this line again?

Reply:
This makes typescript recognize that "window" exists

Reply:
i know, but i would like to avoid it (to not pollute intellisense with dom related stuff (this library is still not "offically" supporting client-side))

Reply:
trying out if i can get around it with optional chaining